pkg/storage/stores/shipper/compactor/deletion/delete_request.go

```diff
@@ -97,13 +97,30 @@ func (d *DeleteRequest) IsDeleted(entry retention.ChunkEntry) (bool, []retention
 		return false, nil
 	}
 
+	level.Debug(util_log.Logger).Log(
+		"msg", "starting filter function",
+		"delete_request_id", d.RequestID,
+		"userID", d.UserID, "labels",
```

Comment:
`<nit>`

```suggestion
		"user", d.UserID, "labels",
```

`</nit>`

We have far more logs with the key "user" than "userID" and it'd be great to keep some consistency here.

```bash
loki [main●●] $ grep '"userID"' -rn pkg | grep log | wc -l
8
loki [main●●] $ grep '"user"' -rn pkg | grep log | wc -l  
43
```

Reply:
Good idea. This was changed manually on the 3 places where userID was used.